docs/_data/menu-sql.yaml

```diff
@@ -71,7 +71,7 @@
 - text: PySpark Usage Guide for Pandas with Apache Arrow
   url: sql-pyspark-pandas-with-arrow.html
 - text: Migration Guide
-  url: sql-migration-old.html
+  url: sql-migration-guide.html
```

Comment:
Do you happen to know the history?

Reply:
cc @HyukjinKwon

Reply:
It was to keep the old URL so existing links are not broken. I think it's been pretty a while so I think we can just remove.